services/galaxytoolshed/galaxytoolshed-version.service.js

```diff
@@ -0,0 +1,85 @@
+import { NotFound } from '../index.js'
+import { renderVersionBadge } from '../version.js'
+import GalaxyToolshedService from './galaxytoolshed-base.js'
+
+export class GalaxyToolshedVersion extends GalaxyToolshedService {
+  static category = 'version'
+  static route = {
+    base: 'galaxytoolshed/v',
+    pattern: ':repository/:owner/:tool?/:requirement?',
+  }
+
+  static examples = [
+    {
+      title: 'Galaxy Toolshed (repository)',
+      pattern: ':repository/:owner',
+      namedParams: {
+        repository: 'sra_tools',
+        owner: 'iuc',
+      },
+      staticPreview: renderVersionBadge({ version: '83c7d564b128' }),
+    },
+    {
+      title: 'Galaxy Toolshed (tool)',
+      pattern: ':repository/:owner/:tool',
+      namedParams: {
+        repository: 'sra_tools',
+        owner: 'iuc',
+        tool: 'fastq_dump',
+      },
+      staticPreview: renderVersionBadge({ version: '1.2.5' }),
+    },
+    {
+      title: 'Galaxy Toolshed (requirement)',
+      pattern: ':repository/:owner/:tool/:requirement',
+      namedParams: {
+        repository: 'sra_tools',
+        owner: 'iuc',
+        tool: 'fastq_dump',
+        requirement: 'perl',
+      },
+      staticPreview: renderVersionBadge({ version: '5.18.1' }),
+    },
+  ]
+
+  static transform({ response, tool, requirement }) {
+    if (tool !== undefined) {
+      const dataTool = response[1].valid_tools.filter(function (x) {
```

Comment:
We generally use the less verbose `x => x.id === tool` anonymous form for these functions, could you please update them?

Reply:
Simplifications of filtering and conditional tests were done in `transform()`